connector_amazon/models/product.py

```diff
@@ -30,15 +30,21 @@ class AmazonProduct(models.Model):
     _description = 'Amazon Product'
 
     record_id = fields.Many2one(
-        'product.product',
-        'Product')
-    external_id = fields.Char(required=True)
+        comodel_name='product.product', string='Product', required=True,
+        ondelete='cascade')
+    external_id = fields.Char(
+        string='SKU',
+        help="Code/sku of the product in the marketplace")
```

Comment:
Nitpck, I suggest everywhere to write sku in capital letters e.g. error logs and etc.